src/app/models/evt-models.ts

```diff
@@ -137,16 +137,15 @@ export interface WitnessGroup {
 }
 export class ApparatusEntry extends GenericElement {
     id: string;
-    content: Reading[];
+    lemma: Reading[];
```

Comment:
Can the lemma have more than one reading?

Reply:
I think no, at least at first level